test/unit/test_NFTestAssert.py

```diff
@@ -101,3 +101,26 @@ def test_assert_expected(mock_assert):
     mock_assert.return_value.get_assert_method = lambda: lambda x, y: False
     with pytest.raises(AssertionError):
         NFTestAssert.assert_expected(mock_assert())
+
+@pytest.mark.parametrize(
+    'glob_return_value,case_pass',
+    [
+        ([], False),
+        (['a', 'b'], False),
+        (['a'], True)
+    ]
+)
+@mock.patch('glob.glob')
+@mock.patch('nftest.NFTestAssert.NFTestAssert', wraps=NFTestAssert)
+def test_identify_assertions_files(mock_assert, mock_glob, glob_return_value, case_pass):
```

Comment:
I suggest writing a unit test for `resolve_single_path` specifically; the additional test on `resolve_single_path` provides a tighter feedback loop if something is to go wrong. This test can probably be left here.